repo-template/.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+    "recommendations": [
+        "sumneko.lua",
+        "overextended.cfxlua-vscode",
+        "ihyajb.qbcore-code-snippets",
+        "EditorConfig.EditorConfig",
+        "eamodio.gitlens",
+        "GitHub.vscode-pull-request-github"
```

Comment:
What are these three? They sound like they may be a personal set preference and therefore maybe shouldn't be on here.

Reply:
Recently a `.editorconfig` was added to all repos so we should keep it. GitLens is pretty good for any git-related features, but I guess it's optional. The pull request one can probably be removed.

Reply:
I copied straight from the core so idk i thought it was something agreed upon already 🤷🏻

Reply:
I think we just remove pull request and gitlens. I use git on command line and many others use the GitHub application directly.